host/CMakeLists.txt

```diff
@@ -298,15 +314,17 @@ set(UHD_BOOST_REQUIRED_COMPONENTS
 
 include(UHDBoost)
 
-include_directories(${Boost_INCLUDE_DIRS})
+include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
```

Comment:
Can you add a comment to describe the goal of including directoy "SYSTEM". May be do int in an include line  dedicated to it.

Reply:
Added a comment here